deploy-board/deploy_board/templates/clusters/base_images.html

```diff
@@ -168,4 +177,60 @@ <h4 class="modal-title">Create Base Image Confirm</h4>
         </div>
     </div>
 </div>
+
+<script>
+    $("#createBaseImageFormId").submit(function(event) {            
+        var provider = $("#providerNameId").val();
+        var errorElement = $("#create-base-image-form-errors");  
+        var errors = "";
+        var validationErrors = [validateProviderName(provider), validateCellName(provider)];
+        validationErrors
+            .filter(err => err != null)
+            .forEach((error, i) => {
+                errors += createValidationErrorHtml(i === 0, "Validation error: " + error);
+            });
+
+        if (errors.length === 0) {
```

Comment:
falsify errror?

Reply:
@ntascii I didn't get, can you please explain your comment?

Reply:
@ntascii `errors.length === 0` uses 3 `=`, because this is JS strict check for equation - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality